modules/lang-mustache/src/test/java/org/elasticsearch/script/mustache/TemplateQueryBuilderTests.java

```diff
@@ -129,9 +129,9 @@ public void testUnknownField() throws IOException {
         String queryAsString = testQueryAsString.replace("inline", "bogusField");
         try {
             parseQuery(queryAsString);
-            fail("ScriptParseException expected.");
-        } catch (ParsingException e) {
-            assertTrue(e.getMessage().contains("unexpected field [bogusField], expected [inline, stored, file, lang, params]"));
+            fail("IllegalArgumentException expected");
+        } catch (IllegalArgumentException e) {
+            assertTrue(e.getMessage().contains("[Script] unknown field [bogusField], parser not found"));
```

Comment:
This is much better written as `assertThat(e.getMessage(), containsString("..."));` so you'll get the message if it fails rather than just "expected true but got false".